docs_src/query_params_str_validations/tutorial006d_an_py39.py

```diff
@@ -1,13 +1,12 @@
 from typing import Annotated
 
 from fastapi import FastAPI, Query
-from pydantic import Required
 
 app = FastAPI()
 
 
 @app.get("/items/")
-async def read_items(q: Annotated[str, Query(min_length=3)] = Required):
+async def read_items(q: Annotated[str, Query(min_length=3)] = ...):
```

Comment:
This has been deleted from the (English) docs in PR #10469, but the Russian documentation does in fact still reference it in `docs/ru/tutorial/query-params-str-validations.md`. Ideally, we probably should remove the reference from the Russian docs as well, and then remove this file altogether, but I would suggest to do that in a separate PR to avoid mixing up different issues - it's always easier to review atomic PRs.